docs/config/lua/config/font_rules.md

```diff
@@ -7,7 +7,7 @@ fonts that you'd like to use.  The default is to auto-generate reasonable
 italic and bold styling based on the [font](font.md) configuration.
 
 If you do specify `font_rules`, they are applied in the order that they are
-specified in the configuration file, stopping with the first matching rule:
+specified, serving all non-negative partial matches.
```

Comment:
I've no clue what this sentence means: what is "serving", what is a "partial match" :-)

Reply:
If you match italic = true, as the first font_rules, it will match italic = true, and all intensities = ' ' unless defined to allow negative match. Tat is, partial meant to mean a larger set of styles the rule is applied to. Italics = true, is a partial match for all intensities.

Reply:
So you want to make clearer what "stopping with the first matching rule" means in practise?

My suggestion would be something like:

```
... specified in the configuration file, stopping with the first matching rule. For that reason, it is recommended to specify rules from most specific to least specific, as e.g. a earlier rule which only specifies `italic = true` will not allow a rule with ` italic = true, intensity = "Normal"` to ever match.
```

Or do I still misunderstand your intend? I still do not understand what `defined to allow negative match` means.

Reply:
That's exactly right, happy with your proposal. As clarity involves what is assumed about the data and rule structure. Assuming that it first tries to match with the most rules first is one of those, for the strength and specificity not mentioned.